dev-vcs/git-lfs/git-lfs-9999.ebuild

```diff
@@ -28,11 +28,11 @@ else
 	#	--mtime="1970-01-01" --sort=name --owner=portage --group=portage
 	# xz -k -9eT0 --memlimit-decompress=256M $P-deps.tar
 	SRC_URI+=" https://files.holgersson.xyz/gentoo/distfiles/golang-pkg-deps/${P}-deps.tar.xz"
-	KEYWORDS="~amd64 ~arm64 ~ppc64 ~riscv ~x86 ~amd64-linux ~x86-linux"
 fi
 
 LICENSE="Apache-2.0 BSD BSD-2 BSD-4 ISC MIT"
 SLOT="0"
+#KEYWORDS="~amd64 ~arm64 ~ppc64 ~riscv ~x86 ~amd64-linux ~x86-linux"
```

Comment:
Moving it outside of the block is unfortunate in that you can't just cp between the ebuilds anymore. Any reason to do that...?

Reply:
In the past tooling had issues (now-obsolete repoman? ekeyword? not sure out of my mind), other maintainers added secondary `KEYWORDS` lines not recognizing the conditional one and similar things, so I started to move the keyword into the unconditional part for ebuilds that I touch regularly a while ago.

Beside that I had ebuilds with three conditions for `SRC_URI`: live, proper releases and custom snapshots (`foo-1.2.3_p20241201.ebuild`). If I've got something like that I typically build something as `if live, else if _p; else do normal fi`.
This would lead to two different `KEYWORDS` lines by design.

So to me `KEYWORDS` is like the `COMMIT_ID` here for this go package something that is just unique to every version of a package and must be just reviewed before commit.

Reply:
I'm not sure if you're misremembering it or if there was a different problem a very long time ago, but it certainly shouldn't be the issue now, or for quite some time: https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0105.

That is, multiple `KEYWORDS` lines are banned.